_includes/header.html

```diff
@@ -7,14 +7,18 @@
 {%- endif -%}
     <div class="main">
       <div class="header__title">
+        {%- assign _paths_root = site.paths.root | default: site.data.variables.default.paths.root  -%}
+        {%- include snippets/get-nav-url.html path=_paths_root -%}
+        <a href="{{ __return }}">
```

Comment:
We may want to still include a title for accessibility purposes. I left out the check for site.title though because we have know we have one.
```suggestion
        <a href="{{ __return }}" title="{%- if site.description -%}{{ site.description }}{%- endif -%}" href="{{ __return }}">
```
```